pkg/pillar/cmd/zedagent/parseconfig.go

```diff
@@ -903,6 +1138,14 @@ func parseDeviceIoListConfig(config *zconfig.EdgeDevConfig,
 			// for SystemAdapter
 			port.UsagePolicy.FreeUplink = ioDevicePtr.UsagePolicy.FreeUplink
 		}
+		if port.Logicallabel == "" {
+			log.Warnf("PhysicalIO without logical label: %+v", port)
```

Comment:
Should we put something like XXX or something else to make all of these easier to find so we can see where we have models which need to be updated?

Reply:
Added XXX